src/Rover.cpp

```diff
@@ -129,6 +129,7 @@ const double CONTROL_HZ = 10.0;
 int rover_loop(int argc, char **argv)
 {
     LOG_LEVEL = LOG_INFO;
+    Globals::AUTONOMOUS = true;
```

Comment:
Just curious, do we have a way to change this while the program is running?

Reply:
Not yet. We do need to implement that (very) soon